main_test.go

```diff
@@ -4,41 +4,63 @@ package gowsl_test
 
 import (
 	"context"
+	"log"
 	"os"
+	"path/filepath"
 	"testing"
 
-	"github.com/0xrawsec/golang-utils/log"
 	wsl "github.com/ubuntu/gowsl"
 	"github.com/ubuntu/gowsl/mock"
 )
 
 const (
-	namePrefix  string = "wsltesting"
-	emptyRootFs string = `./images/empty.tar.gz`  // Empty non-functional image. It registers instantly.
-	rootFs      string = `./images/rootfs.tar.gz` // Fully functional rootfs
+	namePrefix string = "wsltesting"
+)
+
+var (
+	emptyRootFS = `images/empty.tar.gz`  // Empty non-functional image. It registers instantly.
+	rootFS      = `images/rootfs.tar.gz` // Fully functional rootfs
 )
 
 func TestMain(m *testing.M) {
 	ctx := context.Background()
 	if wsl.MockAvailable() {
 		ctx = wsl.WithMock(ctx, mock.New())
+
+		// Touch rootfs so that tests work.
+		// For the real tests, .\prepare-repository.ps1 should be ran before running the tests
+		tmp := filepath.Join(os.TempDir(), filepath.Dir(rootFS))
+		if err := os.MkdirAll(tmp, 0700); err != nil {
+			log.Fatalf("Setup: could not create images dir: %v", err)
+		}
+		defer os.RemoveAll(tmp)
+
+		rootFS = filepath.Join(tmp, rootFS)
+		emptyRootFS = filepath.Join(tmp, emptyRootFS)
+
+		for _, fname := range []string{rootFS, emptyRootFS} {
+			f, err := os.OpenFile(fname, os.O_RDONLY|os.O_CREATE, 0600)
```

Comment:
even if that doesn’t bring much more, but any reason to not use `os.Create` here, which is semantically closer?